model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -70,7 +69,7 @@ def qat_wrapper(n: common.BaseNode, layer: Layer, qat_config):
 
         """
         if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
-            return qi.KerasQuantizationWrapper(layer, quantization_dispatcher_builder(n, qat_config, DEFAULT_KERAS_INFO))
+            return qi.KerasQuantizationWrapper(layer, *quantization_builder(n, qat_config, DEFAULT_KERAS_INFO))
```

Comment:
Why not taking the two arguments from the pair and pass them explicitly? the length of the number of the arguments you pass to KerasQuantizationWrapper is always 2, no need to use collection unfolding. I think it is clearer to pass the arguments explicitly.